bin/omakub-theme

```diff
@@ -1,39 +1,29 @@
 # Pick a preconfigured theme
 THEME_NAMES=("Tokyo Night" "Catppuccin" "Nord" "Everforest" "Gruvbox" "Kanagawa" "Rose Pine")
+
 THEME=$(gum choose "${THEME_NAMES[@]}" --header "Choose your theme" --height 9 | tr '[:upper:]' '[:lower:]' | sed 's/ /-/g')
 
-[ ! -n "$THEME" ] && exit 0
+[ -z "$THEME" ] && exit 0
 
 # Install theme in Gnome, Terminal, and both default editors
-source $OMAKUB_PATH/themes/gnome/$THEME.sh
-cp $OMAKUB_PATH/themes/alacritty/$THEME.toml ~/.config/alacritty/theme.toml
-cp $OMAKUB_PATH/themes/neovim/$THEME.lua ~/.config/nvim/lua/plugins/theme.lua
+source "$OMAKUB_PATH/themes/gnome/$THEME.sh"
+cp "$OMAKUB_PATH/themes/alacritty/$THEME.toml" ~/.config/alacritty/theme.toml
+cp "$OMAKUB_PATH/themes/neovim/$THEME.lua" ~/.config/nvim/lua/plugins/theme.lua
```

Comment:
don't see why we need this quoting? The value of $THEME is known to be safe.

Reply:
yep my bad, it's a bad habit I got